third_party/move/move-vm/runtime/src/storage/publishing.rs

```diff
@@ -52,7 +52,7 @@ impl<K: Ord, V: Clone> IntoIterator for VerifiedModuleBundle<K, V> {
 struct StagingModuleBytesStorage<'a, M> {
     staged_runtime_environment: RuntimeEnvironment,
     // Modules to be published, staged temporarily.
-    staged_module_bytes: BTreeMap<AccountAddress, BTreeMap<Identifier, Bytes>>,
+    staged_modules: BTreeMap<AccountAddress, BTreeMap<Identifier, (Bytes, Arc<CompiledModule>)>>,
```

Comment:
would an Rc not work? we can't avoid a clone to creating the locally verified module I assume (or else we would have just stored CompiledModule directly?)